Monika After Story/game/script-songs.rpy

```diff
@@ -158,6 +217,49 @@ label monika_sing_song_pool_menu:
 
     return
 
+#START: Song analysis delegate
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_sing_song_analysis",
+            prompt="Can you explain a song for me?",
+            category=["music"],
+            pool=True,
+            unlocked=False,
+            aff_range=(mas_aff.NORMAL, None),
+            rules={"no unlock": None}
+        )
+    )
+
+
+label monika_sing_song_analysis:
+    python:
+        ret_back = ("Nevermind.", False, False, False, 20)
+
+        unlocked_analyses = mas_songs.getUnlockedSongAnalyses()
+
+        if mas_isO31():
+            which = "Witch"
+        else:
+            which = "Which"
+
+    show monika 1eua at t21
+    $ renpy.say(m, "[which] song would you like me to sing?", interact=False)
```

Comment:
Will she sing songs here too or will she just explain them? Because if it's the latter, then `sing` doesn't work here. Might use this?
```suggestion
    $ renpy.say(m, "[which] song would you like to talk about?", interact=False)
```

Reply:
Oops, copy/paste lol